packages/components-react/src/SideNavigation/index.tsx

```diff
@@ -0,0 +1,87 @@
+import { IconArrowBarToLeft, IconArrowBarToRight, IconChevronDown, IconChevronUp } from '@tabler/icons-react';
+import { Button, Link } from '@utrecht/component-library-react/dist/css-module';
+import clsx from 'clsx';
+import { HTMLAttributes, PropsWithChildren, useState } from 'react';
+import '@nl-design-system-kernteam/components-css/side-navigation/index.scss';
+
+export interface SideNavigationProps extends HTMLAttributes<HTMLElement> {
+  listItems: MenuListItemProps[];
+}
+
+export const SideNavigation = ({ className, listItems, ...restProps }: SideNavigationProps) => {
+  const [open, setOpen] = useState(true);
+
+  const sideNavClassname = clsx(className, 'kernteam-side-navigation', {
+    ['kernteam-side-navigation--hidden']: !open,
+  });
+
+  return (
+    <div className={sideNavClassname} {...restProps}>
+      <Button
+        className={'kernteam-side-navigation__toggle-button'}
+        appearance={'subtle-button'}
+        onClick={() => setOpen(!open)}
+      >
+        {open ? (
+          <>
+            <IconArrowBarToLeft />
```

Comment:
laten we het icon visueel verberg en als presentational markeren

Reply:
Dat kan met de <Icon> component van Utrecht